src/sentry/tasks/assemble.py

```diff
@@ -364,6 +364,13 @@ def _create_artifact_bundle(
     with ReleaseArchive(archive_file.getfile()) as archive:
         bundle_id, debug_ids_with_types = _extract_debug_ids_from_manifest(archive.manifest)
 
+        analytics.record(
+            "artifactbundle.assemble_complete",
```

Comment:
Technically here the assemble didn't complete, it could also be that we have failures inserting into the db. I would rather call it:
```suggestion
            "artifactbundle.manifest_extracted",
```